### DISCUSSION TITLE
Package or Module Support?

### DISCUSSION THREAD

**Discussion by curiousblueprints** [twentyhq/twenty#8917] [Ideas]
Most CRM services have some sort of add-on or module capability to install third-party features. This can be done through a storefront (eg Zoho Marketplace) or through APIs or other features (eg “Deploy to Salesforce” buttons on some SF GitHub package repos.)

Typically a package consists of metadata such as custom fields, custom logic, integrations, and more. In most services, these items are namespaced to minimize conflicts between packages and between a package and any customizations built directly in the CRM.

Has there been any thought given to 3rd-Party module support in any way? Is this of interest to the Twenty Team?

**Comment by BOHEUS** [comment]
Currently core team is working on extensibility, SDK is more mature than before but it'll take a bit of time till it's stable but definitely it'll be released this year (most likely this quarter), you can read more about it here https://docs.twenty.com/developers/extend/apps/getting-started
